qualtran/bloqs/mean_estimation/mean_estimation_operator_test.py

```diff
@@ -63,7 +62,7 @@ class BernoulliEncoder(SelectOracle, SpecializedSingleQubitControlledExtension):
 
     @cached_property
     def control_registers(self) -> Tuple[Register, ...]:
-        return () if self.control_val is None else (Register('control', QBit()),)
+        return ()
```

Comment:
Do we need to keep around this property if it's just empty? (I missed the older PRs so probably need to read those)

Reply:
`SelectOracle` has this as an abstract property, but as there's no special controlled version I just return `()`.

Ideally the select/prepare interfaces should be updated to the newer `BlockEncoding` style interfaces (would be a major refactor), but as of now as callees use `.control_registers`, I'm keeping it as-is.